src/main/java/org/gradle/profiler/InvocationSettings.java

```diff
@@ -164,6 +167,10 @@ public Format getCsvFormat() {
         return csvFormat;
     }
 
+    public String getTitle() {
```

Comment:
I think it would be good to have some Javadoc in this method which also helps distinguish between `InvocationSettings.title` and `ScenarioDefinition.title`.

Reply:
I opted to rename it to `benchmarkTitle` instead. WDYT?